components/GardenPage/GardenPage.js

```diff
@@ -0,0 +1,379 @@
+`;
+
+const PetWrapper = styled.div`
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const PetDisplay = styled.div`
+  font-size: 8em;
+  color: var(--text-color);
+  transform-origin: center;
+  animation: ${({ $rotating, $bouncing, $growing }) =>
+    $rotating
+      ? css`
+          ${rotate} 1s linear
+        `
+      : $bouncing
+      ? css`
+          ${bounce} 1s ease
+        `
+      : $growing
+      ? css`
+          ${grow} 0.5s ease
+        `
+      : "none"};
+`;
+
+const NavbarContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px;
+  background-color: var(--neutral-color);
+  border-radius: 8px;
+  margin: auto;
+  max-width: 600px;
+`;
+
+const NavButton = styled.button`
+  background-color: var(--primary-color);
+  color: var(--text-color);
+  border: none;
+  padding: 10px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px;
+  cursor: pointer;
+  border-radius: 4px;
+`;
+
+const StyledLink = styled.div`
+  background-color: var(--signal-color);
+  color: var(--text-color);
+  padding: 10px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px;
+  cursor: pointer;
+  border-radius: 4px;
+`;
+
+const StatusContainer = styled.div`
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`;
+
+const VerticalBarContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  margin-top: 10px;
+`;
+
+const VerticalBar = styled.div`
+  width: 20px;
+  height: 100px;
+  background-color: var(--neutral-color);
+  margin-right: 5px;
+  border-radius: 4px;
+  overflow: hidden;
+  position: relative;
+`;
+
+const VerticalBarFill = styled.div`
+  width: 100%;
+  background-color: ${(props) => props.$bgcolor};
+  height: ${({ value }) => value}%;
+  position: absolute;
+  bottom: 0;
+  z-index: 1;
+`;
+
+const Icon = styled.span`
+  z-index: 2;
+  position: absolute;
+`;
+
+const HorizontalBar = styled.div`
+  width: 100px;
+  height: 20px;
+  background-color: var(--neutral-color);
+  border-radius: 4px;
+  margin-bottom: 10px;
+  overflow: hidden;
+  position: relative;
+`;
+
+const HorizontalBarFill = styled.div`
+  width: ${({ value }) => value}%;
+  height: 100%;
+  background-color: green; //color is a placeholder
+  position: absolute;
+  left: 0;
+`;
+
+const ButtonContainer = styled.div`
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+`;
+
+const StatusButton = styled.button`
+  background-color: ${(props) => props.$bgcolor};
+  color: var(--text-color);
+  border: none;
+  padding: 16px;
+  margin-bottom: 8px;
+  border-radius: 4px;
+  cursor: pointer;
+  width: 75px;
+`;
+
+const ListPageLink = styled.div`
```

Comment:
I was under the impression this would make reading the code easier to keep it out of the return section.
And also to keep it consistent to the othe rlink